### ISSUE TITLE
[BUG] Pages navigation at end of Posts display wrong direction.

### ISSUE THREAD

**Issue by GreatestOfAllTimee** [HEIGE-PCloud/DoIt#476]
### Describe the bug 描述你遇到的错误:

Thank you for maintaining this Hugo theme. What a wonderful work! 🙏
Actually I don't know where to start, here are example images down below.

#### 1. **Series Image:**
![Series1](https://user-images.githubusercontent.com/73665622/155136875-4e730ff9-987e-4a47-aff0-f2e3d1cc354b.png)
```markdown
<!--frontmatter -->
weight: 1
series_weight: 1
...
```

#### 2. **Button navigation to next posts:**
![NextPages](https://user-images.githubusercontent.com/73665622/155137140-7cecf930-5ff6-492b-a977-7ceeedd759a8.png)
```markdown
<!--frontmatter-->
weight: 2
series_weight: 2
...
```
---

According to first image "[Theme Documentation - Basics](https://do-it-pcloud.vercel.app/theme-documentation-basics)" are the **first Series (_series_weight: 1_)**, if we look at the second image, navigate button to next series/posts are on the _left_. Expected behavior describe down below.

### Expected to Happen:

Button navigation (on the end of the page (like the second image above ) to next series should on the right site -- unless support by specific language. OR maybe i missing something? 

### Build Environment 构建环境

- Images above was screenshot from https://do-it-pcloud.vercel.app & https://hugodoit.com

Sorry for my poor explanation, caused English is not my mother tongue.

**Comment by GreatestOfAllTimee** [comment]
Yes I get this. But still confuse how Loveit/DoIt themes navigate to next page. Is it by date? Not weight?

**Comment by HEIGE-PCloud** [comment]
The order is Weight > Date > LinkTitle > FilePath. If a post has a higher weight, then it will be the "next" post. If the weights are the same, then the post with the latest date will be the next post, and so on... As I mentioned above, the order should be the same as the order of the posts on the home page.
